From 8cfc017a277b2bfe860631502c23cb8f985604a8 Mon Sep 17 00:00:00 2001
From: Atchaya R <quic_ratchaya@quicinc.com>
Date: Tue, 1 Aug 2023 17:15:23 +0530
Subject: [PATCH] certs: Add -lpthread to HOSTLDLIBS_extract-cert

On Ubuntu 18.04 distributions, libpthread is not
automatically linked.
This results in unresolved symbols, added -lpthread
to link libpthread with extract-cert.

Change-Id: I876669a9d35085b2da312036a1f0d0ad93a12eb3
Signed-off-by: Atchaya R <quic_ratchaya@quicinc.com>
---
 certs/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/certs/Makefile b/certs/Makefile
index 799ad7b9e68a..2679a77a739f 100644
--- a/certs/Makefile
+++ b/certs/Makefile
@@ -85,4 +85,4 @@ targets += x509_revocation_list
 hostprogs := extract-cert
 
 HOSTCFLAGS_extract-cert.o = $(shell $(HOSTPKG_CONFIG) --cflags libcrypto 2> /dev/null)
-HOSTLDLIBS_extract-cert = $(shell $(HOSTPKG_CONFIG) --libs libcrypto 2> /dev/null || echo -lcrypto)
+HOSTLDLIBS_extract-cert = $(shell $(HOSTPKG_CONFIG) --libs libcrypto 2> /dev/null || echo -lcrypto) -lpthread
-- 
2.34.1

